### DISCUSSION TITLE
Definition/threshold for full-length counts reported by Bambu

### DISCUSSION THREAD

**Discussion by bernardo-heberle** [GoekeLab/bambu#379] [Q&A] [answered]
Hi,

I could not find a definition for what Bambu reports as a full-length count/read. Is there a specific threshold?

For example: 

Full-length count = A read that covers > 95% of a transcript then it is considered full-length.

**Comment by andredsim** [comment, marked as answer]
Hi,

A read is considered full length if all intron chains are present. We do this because it is not always clear what the true start and end of the transcript is (because of degradation and multiple transcription start-sites) but we can be confident about splice junctions.
